user-interface/frontend/themes/datamanager/components/all.css

```diff
@@ -1,8 +1,332 @@
+/****************************************
+
+Data Manager custom CSS layout definition
+
+*****************************************/
+
+/*
+First some general property definitions
+*/
+:root {
+  --dialog-height-small: clamp(min-content, 50vh, 75vh);
+  --dialog-height-medium: clamp(min-content, 75vh, 100vh);
+  --dialog-height-large: 100vh;
+  --dialog-height-max-small: 500px;
+  --dialog-height-max-medium: 800px;
+  --dialog-height-max-large: 100vh;
+  --dialog-width-small: 50vw;
+  --dialog-width-medium: 75vw;
+  --dialog-width-large: 80vw;
+  --dialog-width-max-small: 600px;
+  --dialog-width-max-medium: 1000px;
+  --dialog-width-max-large: 1200px;
+  --font-weight-light: 300;
+  --font-weight-regular: 400;
+  --font-weight-medium: 500;
+  --font-weight-semi-bold: 600;
+  --font-weight-bold: 700;
+  --icon-color-info: rgba(22, 118, 243, 1);
+  --icon-color-error: rgba(255, 66, 56, 1);
+  --icon-color-success: rgba(21, 193, 93, 1);
+  --icon-color-warning: rgba(254, 201, 1, 1);
```

Comment:
any relation to `--lumo-warning-color: hsl(48,100%,50%,0.25)`? Are the warning, success and error colors dependant on the `--lumo-error-color` and `--lumo-success-color`?

Reply:
irrelevant, defined by design. Just following the style guide here.